selftests/unit/test_nrunner.py

```diff
@@ -191,7 +191,7 @@ def test_runner_noop(self):
         results = [status for status in runner.run()]
         last_result = results[-1]
         self.assertEqual(last_result['status'], 'finished')
-        self.assertIn('time_end', last_result)
+        self.assertIn('time', last_result)
```

Comment:
These (4 in total) don't belong in this commit.

Reply:
You are right, fixed in v2.